### DISCUSSION TITLE
How to use dinit instead of sysvinit?

### DISCUSSION THREAD

**Discussion by zoujiaqing** [davmac314/dinit#161] [Q&A] [answered]
I have a FreeBSD 14 computer.
What should I do?

**Comment by davmac314** [comment, marked as answer]
From the README:
> Dinit can run as the system "init" - the first process started by the kernel on boot - which is normally done by linking or copying dinit to /sbin/init. This is currently supported only on Linux. It requires having suitable service descriptions in place and should be attempted only by those comfortable with low-level system administration and recovery. See [doc/linux](https://github.com/davmac314/dinit/blob/master/doc/linux) directory for more information.

**Comment by mobin-2008** [comment]
Hi.
1. FreeBSD doesn't use `sysvinit`. Most BSDs have a custom init. init of FreeBSD [Source](https://github.com/freebsd/freebsd-src/blob/main/sbin/init/init.c)
2. Dinit is designed to work under POSIX compatible systems but in practice current Dinit can't boot FreeBSD (due to some technical things). There is ToDo for ver 1.0
3. You need suitable service descriptions for booting FreeBSD correctly.

Regards

**Comment by Konpaku-LVX** [comment]
i just want to use bsd without waiting 30 seconds to boot. linux can boot in 5 seconds with modern init software.
